### ISSUE TITLE
Jobspec file exception error management

### ISSUE THREAD

**Issue by cmisale** [flux-framework/flux-sched#473]
In my experiments I ran into `ERROR: Exception occurs for input file I/Obasic_ios::clear` error when trying to open a jobspec on a wrong file path.
It would be useful to have a better exception management with specific error messages.
Lines issuing that exception are the following
https://github.com/flux-framework/flux-sched/blob/a3dcb3d5dc72a9338f5c7af8f40e1ee06912f38a/resource/utilities/command.cpp#L173
raising after
https://github.com/flux-framework/flux-sched/blob/a3dcb3d5dc72a9338f5c7af8f40e1ee06912f38a/resource/utilities/command.cpp#L144

**Comment by dongahn** [comment]
Thanks @cmisale!

**Comment by dongahn** [comment]
This was fixed by PR #543 

```console
ahn1@9322372c5ba3:/usr/src/resource/utilities$ ./resource-query -L ../../t/data/resource/grugs/tiny.graphml
INFO: Loading a matcher: CA
resource-query>
resource-query>
resource-query> match allocate test_no.yaml
ERROR: can't open test_no.yaml
resource-query> update allocate test_no 1 1 1000
ERROR: can't open test_no
